test/dialing/direct.spec.ts

```diff
@@ -434,11 +434,9 @@ describe('libp2p.dialer (direct, WebSockets)', () => {
 
     await libp2p.start()
 
-    libp2p.components.getUpgrader().addEventListener('connection', () => {
+    libp2p.connectionManager.addEventListener('peer:connect', () => {
```

Comment:
Please use `pEvent` here, it's more concise and better reflects the intention behind the code.